src/components/index.esm.js

```diff
@@ -41,11 +41,18 @@ export * from './pagination-nav'
 export * from './popover'
 export * from './progress'
 export * from './spinner'
-export * from './table'
+// Webpack 4 has difficulties with re-eport of re-export optimizations
```

Comment:
`re-export of re-exports`

Reply:
Oops... was supposed to be commented out.

Reply:
Yeah... structured exports exporting from the net level.